catalog/resource_external_location.go

```diff
@@ -49,6 +45,8 @@ func ResourceExternalLocation() *schema.Resource {
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
 			return m
 		})
+	updateOwner := updateFunctionFactory("/unity-catalog/external-locations/", []string{"owner"})
+	update := updateFunctionFactory("/unity-catalog/external-locations/", []string{"owner", "name", "comment", "url", "credential_name", "skip_validation"})
```

Comment:
ouch, this looks ugly. there's `d.IsNewResource()` that can probably help to diff between create and update. you can add a special if statement in update factory just for `owner` field (if it's present in the list of fields).

i don't remember if it's easy unit testable or not, so please launch provider in a debug mode to see how to use it and then work on tests :)